<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>781</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Анализ вебинаров</string>
  </property>
  <widget class="QPushButton" name="nextButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>720</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="windows">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>1180</width>
     <height>500</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="window_1">
    <attribute name="title">
     <string>Этап 1</string>
    </attribute>
    <widget class="QPushButton" name="extractSilenceButton">
     <property name="geometry">
      <rect>
       <x>201</x>
       <y>410</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Извлечь интервалы тишины</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="settingsParameters_groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>500</width>
       <height>220</height>
      </rect>
     </property>
     <property name="title">
      <string>Установка параметров</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Разбить по : </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="settingsParameters_splitMs">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>миллисекунд</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsParameters_setButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>170</y>
        <width>131</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Установить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>82</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Допустимая пауза в : </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="settingsParameters_maxSilenceMs">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>84</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>85</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>миллисекунд</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Минимальная длина фреймов :</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="settingsParameters_minLengthFrameMs">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>миллисекунд</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="loadVideo_groupBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>991</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Загрузка видео</string>
     </property>
     <widget class="QToolButton" name="loadVideo_pathButton">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>40</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loadVideo_pathEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>781</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Путь к вебинару</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadVideo_loadButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>110</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Загрузить видео</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="filterButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>180</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Отфильтровать</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="statusProcesses_groupBox">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>190</y>
       <width>651</width>
       <height>270</height>
      </rect>
     </property>
     <property name="title">
      <string>Статус процессов</string>
     </property>
     <widget class="QTextEdit" name="textBox_status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>631</width>
        <height>230</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="window_2">
    <attribute name="title">
     <string>Этап 2</string>
    </attribute>
    <widget class="QGroupBox" name="statusProcesses_groupBox_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>190</y>
       <width>650</width>
       <height>280</height>
      </rect>
     </property>
     <property name="title">
      <string>Статус процессов</string>
     </property>
     <widget class="QTextEdit" name="textBox_status_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>630</width>
        <height>240</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="typeWork_groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>310</width>
       <height>160</height>
      </rect>
     </property>
     <property name="title">
      <string>Режимы работы</string>
     </property>
     <widget class="QRadioButton" name="manuallyWork_radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>300</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ручной режим</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="semiautomaticWork_radioButton">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>80</y>
        <width>300</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Полуавтоматический режим</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveMode_button">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>110</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Принять</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="trainClassificator_groupBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>820</width>
       <height>110</height>
      </rect>
     </property>
     <property name="title">
      <string>Интервал голоса ведущего для классификатора</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>53</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>От</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>53</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>До</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trainClassificator_button">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>43</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Тренировать</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="fromIntervalPresenter_timeEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>HH.mm.ss.zzz</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="toIntervalPresenter_timeEdit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>HH.mm.ss.zzz</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Этап 3</string>
    </attribute>
    <widget class="QGroupBox" name="statusProcesses_groupBox_3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>190</y>
       <width>650</width>
       <height>280</height>
      </rect>
     </property>
     <property name="title">
      <string>Статус процессов</string>
     </property>
     <widget class="QTextEdit" name="textBox_status_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>630</width>
        <height>240</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="setParameters_groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>500</width>
       <height>460</height>
      </rect>
     </property>
     <property name="title">
      <string>Установка параметров</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>210</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Минимальное расстояние</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="minDistance_spinBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>210</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Максимальное расстояние</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="maxDistance_spinBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimum">
       <number>7</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
     <widget class="QPushButton" name="setParameters_button">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>390</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Установить</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="showRegionConference_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>440</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Показывать область конференции во время анализа</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="saveAllIntervalsInFiles_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>400</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить все полученные интервалы в файлы</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>320</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Время между анализируемыми кадрами</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="timeBetweenFrames_spinBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimum">
       <number>25</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>110</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>миллисекунд</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="startVideoAnalyze_button">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Начать</string>
     </property>
    </widget>
    <zorder>startVideoAnalyze_button</zorder>
    <zorder>statusProcesses_groupBox_3</zorder>
    <zorder>setParameters_groupBox</zorder>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tab_intervals">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>1001</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_intervalsSilence">
    <attribute name="title">
     <string>Интервалы тишины</string>
    </attribute>
    <widget class="QTextEdit" name="textBox_intervalsSilence">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>971</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Интервалы еще не выделены</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_intervalsPresenter">
    <attribute name="title">
     <string>Интервалы разговора ведущего</string>
    </attribute>
    <widget class="QTextEdit" name="textBox_intervalsPresenter">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>970</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Интервалы еще не выделены</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_intervalsSomeone">
    <attribute name="title">
     <string>Интервалы разговора участника</string>
    </attribute>
    <widget class="QTextEdit" name="textBox_intervalsSomeone">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>971</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Интервалы еще не выделены</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_intervalsTogether">
    <attribute name="title">
     <string>Интервалы разговора ведущего и участника</string>
    </attribute>
    <widget class="QTextEdit" name="textBox_intervalsTogether">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>971</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Интервалы еще не выделены</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="editingIntervals_pushButton">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>650</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
